emails/templates/emails/wrapped_email.html

```diff
@@ -12,110 +12,155 @@
-                  {{ email_wrap_forwarded_from|safe }}
-                  {% if has_attachment %}
-                      {% ftlmsg 'nav-faq' as faq_text %}
-                      {% with SITE_ORIGIN|add:"/faq/" as faq_url %}
-                          {% bold_violet_link faq_url faq_text as faq_link %}
-                          {% ftlmsg 'forwarded-email-header-attachment' faq_link=faq_link size='10' unit='MB' as forwarded_email_wrap_attachment %}
-                          {{ forwarded_email_wrap_attachment|safe }}
-                      {% endwith %}
-                  {% else %}
-                    {% if survey_text and survey_link %}
-                    <br><a href="{{ survey_link }}" target="_blank" rel="noopener noreferrer" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">{{ survey_text }}</a>.
+          <td align="center" valign="top" width="100%" style="padding-top: 0px; padding-bottom: 0px;">
+          <table width="100%" style="max-width:1200px; text-align: center; border-collapse: collapse;">
+              <tr> 
+                <td class="header-block" width="50%" align="left"> 
+                  <img width="30" src="{{ SITE_ORIGIN }}/static/images/email-images/relay-icon.png" style="margin-right: 5px; display: inline-block; vertical-align: top;" alt="relay icon"/>   
+                   
+                  <p style="margin-top: 0; margin-bottom: 5px; display: inline-block;"> 
+                    {% with display_email|urlencode as mask_url %}
+                    <span style="display: block;"> 
+                    {% ftlmsg 'relay-email-forwarded-from' %} 
+                    <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile/#{{ mask_url }}" style="margin-right: 30px;"> {{ display_email }} </a>
+                    {% endwith %}
+                    </span>
+                
+
+                  <span style="margin-top: 0; margin-bottom: 5px; display: block;">{% ftlmsg 'relay-email-premium-by-line' %} <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile" style="margin-right: 30px;">{% ftlmsg 'relay-email-premium-by-line-link' %}</a>
+                  </span>
+                  </p>  
+                    
+                </td> 
+                <td class="header-block" width="50%" align="right"> 
+                  <p style="margin: 0 30px 0 0; display: inline-block;">
+                    <img width="18" src="{{ SITE_ORIGIN }}/static/images/email-images/email-trackers-removed-icon.png" style="margin-right: 5px; position: relative; top: 5px;" alt="warning icon"/>   
+
+                    {% comment %}
+                      Create this as a link if we have a report link to show 
+                    {% endcomment %}
+                    {% if has_tracker_report_link %}
+                    <a class="container-link" href="{{ tracker_report_link|urlencode  }}"> 
```

Comment:
Blocking: the `tracker_report_link` is already URL-encoded as necessary. URL-encoding it again turns the `#` into `%23`, breaking the link:

```suggestion
                    <a class="container-link" href="{{ tracker_report_link  }}"> 
```